### PR TITLE
Sqs worker

### PR DESCRIPTION
**Pull request by tim-harper**

### CODE REVIEW

File: internal/datastore/discussion.go [hunk 1]
```diff
@@ -105,103 +107,50 @@ func (d *delphisDB) UpsertDiscussion(ctx context.Context, discussion model.Discu
...
+		ctx,
+		tag.ID,
+		tag.Tag,
+	).Scan(
+		&tag.ID,
+		&tag.Tag,
+		&tag.CreatedAt,
+	); err != nil {
+		if err == sql.ErrNoRows {
+			return &model.Tag{}, nil
```
Comment (tim-harper) [inline internal/datastore/discussion.go:149]: how do we want to handle collisions? this returns null on collisions.

**Comment by tim-harper** [comment]
GQL Call to see what articles of interest are up next for the discussion
image

File: config/local.json [hunk 2]
```diff
@@ -58,5 +58,9 @@
         "image_prefix": "images",
         "gif_prefix": "gifs",
         "video_prefix": "videos"
+    },
+    "sqs": {
+        "drip_url": "https://sqs.us-west-2.amazonaws.com/033236388136/non_terraform_zapier_queue",
+        "max_workers": 2
```
Comment (tim-harper) [inline config/local.json:64]: I can look into a way to mock local resources to replace aws resources

File: internal/backend/auto_post.go [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		if len(contents) == 0 {
+			return nil
+		}
+	}
+
+	// Call post function
+	// TODO: Create concierge participantID to handle posting
+	content := contents[0]
+	now := time.Now()
+	if _, err := d.PostImportedContent(ctx, "073e3c99-8b7e-4548-b94a-1fb94219fc05", discussionID, content.ID, &now, content.Tags); err != nil {
```
Comment (nedrocks) [inline internal/backend/auto_post.go:72]: Can we make this a constant? Also won't it change per environment?

File: internal/datastore/discussion.go (hunk 1 above)
Comment (nedrocks) [reply internal/datastore/discussion.go:169]: In this case a collision would be if a discussion already has the tags that are being added?

File: _scripts/migrations/0013_imported_content.sql [hunk 4]
```diff
@@ -0,0 +1,54 @@
+CREATE TABLE IF NOT EXISTS imported_contents (
```
Comment (nedrocks) [inline _scripts/migrations/0013_imported_content.sql:1]: Adding some test comment

File: _scripts/migrations/0013_imported_content.sql [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+);
+
+CREATE TABLE IF NOT EXISTS imported_content_tags (
+    imported_content_id varchar(36) not null,
+    deleted_at timestamp with time zone,
+    tag varchar(40) not null,
+    created_at timestamp with time zone default current_timestamp not null,
+    PRIMARY KEY(imported_content_id, tag)
+);
+
```
Comment (nedrocks) [inline _scripts/migrations/0013_imported_content.sql:26]: this is another test

File: _scripts/migrations/0013_imported_content.sql [hunk 6]
```diff
@@ -0,0 +1,54 @@
+CREATE TABLE IF NOT EXISTS imported_contents (
+    id varchar(36) PRIMARY KEY,
+    created_at timestamp with time zone default current_timestamp not null,
```
Comment (nedrocks) [inline _scripts/migrations/0013_imported_content.sql:3]: some comment

File: _scripts/migrations/0013_imported_content.sql [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    content_name text not null,
+    content_type text not null,
```
Comment (nedrocks) [inline _scripts/migrations/0013_imported_content.sql:5]: comment 2

**Pull request merged**